# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/composable_kernel/hotfix-profiler-gfx9.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- composable_kernel-rocm-6.3.3/CMakeLists.txt.vanilla	2025-04-01 20:28:03.876477979 +0200
+++ composable_kernel-rocm-6.3.3/CMakeLists.txt	2025-04-01 20:28:44.137071641 +0200
@@ -565,13 +565,13 @@
    endif()
 endif()
 
+if(CK_USE_CODEGEN AND (GPU_TARGETS MATCHES "gfx9" OR GPU_ARCHS))
 rocm_package_setup_component(profiler
     LIBRARY_NAME composablekernel
     PACKAGE_NAME ckprofiler
 )
 add_subdirectory(profiler)
 
-if(CK_USE_CODEGEN AND (GPU_TARGETS MATCHES "gfx9" OR GPU_ARCHS))
   add_subdirectory(codegen)
 endif()
 
